plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlClient.java

```diff
@@ -113,10 +118,17 @@ public class TestPostgreSqlClient
             new JdbcStatisticsConfig(),
             session -> { throw new UnsupportedOperationException(); },
             new DefaultQueryBuilder(RemoteQueryModifier.NONE),
-            TESTING_TYPE_MANAGER,
+            getTestingTypeManager(),
```

Comment:
We could use `new TestingPostgreSqlConnectorContext().getTypeManager()` and remove the new `getTestingTypeManager` method.

Reply:
Done